### PR TITLE
Always install FletchConfig in the Fletch_BUILD_INSTALL_PREFIX dir

### PR DESCRIPTION
**Pull request by vovythevov**
Fletch automatically installs everything to the Fletch_BUILD_INSTALL_PREFIX
directory, but kept the cmake configuration file in the build directory.
Because other projects actually build against the install, the FletchConfig.cmake
should be in the install directory too.

CC: @dstoup

### CODE REVIEW

**Comment by hughed2** [comment]
Can one of the admins verify this patch?

**Comment by hughed2** [comment]
Can one of the admins verify this patch?

File: CMakeLists.txt [hunk 1]
```diff
@@ -175,7 +176,6 @@ set (COMMON_CMAKE_ARGS
   -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
   -DCMAKE_INSTALL_LIBDIR:PATH=${fletch_BUILD_INSTALL_PREFIX}/lib
   -DCMAKE_PREFIX_PATH:PATH=${fletch_BUILD_INSTALL_PREFIX}
-  -DCMAKE_INSTALL_PREFIX:PATH=${fletch_BUILD_INSTALL_PREFIX}
```
Comment (dstoup) [inline CMakeLists.txt]: This removal is the only change I don't understand/agree with. This directive is important for the subprojects that use CMake. Is it causing some issue?

Comment (dstoup) [inline CMakeLists.txt]: This removal is the only change I don't understand/agree with. This directive is important for the subprojects that use CMake. Is it causing some issue?

Comment (vovythevov) [inline CMakeLists.txt]: @dstoup: That was to remove a duplicate line. I separated my commits to make it more clear. Thanks for the review !

**Pull request merged**